### ISSUE TITLE
User avatar box in top menu shows different size according to length of username

### ISSUE THREAD

**Issue by gurjyot** [q2a/question2answer#281]
User avatar box in top menu should be fixed for all users irrespective of username. It looks very awkward that box change its size according to username and mostly it is annoying in mobiles. See these three sizes which we get in mobile view.

![abhi](https://cloud.githubusercontent.com/assets/6462850/9221262/a55b085a-4106-11e5-8916-24b6c3306cee.png)
![gs](https://cloud.githubusercontent.com/assets/6462850/9221263/a5605f26-4106-11e5-9351-232b710e409b.png)
![rk](https://cloud.githubusercontent.com/assets/6462850/9221264/a567162c-4106-11e5-94ec-d30202854d57.png)

**Comment by q2amarket** [comment]
@gurjyot this is obvious thing. When text are long than it will expand the width. If you want it with fix width than set with `overflow: hidden` but it will hide rest of the text. You also can try to use `word-wrap: break-word` but it will look strange.

**Comment by gurjyot** [comment]
I know it is obvious that if there is more text then box size will increase. But this thing is very annoying. So that's why I am asking for fix the box size like the menu button is fixed on the left side. 
What you suggested is good but as you said it'll ruin the looks, that's why I raised this issue to get a solid and permanent solution.

**Comment by svivian** [comment]
Yeah this is a tough one. Possibly we could set a max width with `overflow: ellipsis`.

**Comment by gurjyot** [comment]
Or we can just go back to showing avatar only. Because even clicking on avatar we get to see username then why to have two usernames at the same place.
Moreover, just an avatar would look cool and much clean. Even the admins can set their logo icon as user avatar and that would give a sophisticated look.

**Comment by q2amarket** [comment]
@svivian got good idea after your comment....we can set max-width with `overflow;ellipsis` and also can  include the title attribute with the username to either entire box or just on a username.

Let me know what you think than will modify it..

**Comment by svivian** [comment]
Hi guys, finally looked at this again and decided on a solution. As several people have requested I've removed the username from the menu. Many other sites are doing this nowadays so it should be easy enough for people to use.

I fixed the main issue which was the blank menu when there is no avatar. So it will now show the username on its own if there is no avatar. And it has the `text-overflow: ellipsis` as discussed so that it doesn't mess up the layout.

Thanks to everyone for the feedback and ideas.
